### PR TITLE
Enables debug logging

### PR DESCRIPTION
**Pull request by atreat**
Also moves the project to the Root Enterprise development team and switches the bundle id to accommodate  this.

### CODE REVIEW

File: root-drive-science-ios.xcodeproj/project.pbxproj
Comment (frankmassi-root) [inline root-drive-science-ios.xcodeproj/project.pbxproj:529]: why are the bundle ids changing?

**Review by frankmassi-root — APPROVED** [review]
Seems fine. I'm confused by the bundle id changes, but assuming it's just ios being ios, LGTM.

File: root-drive-science-ios.xcodeproj/project.pbxproj
Comment (atreat) [reply root-drive-science-ios.xcodeproj/project.pbxproj:529]: they are globally unique and I figured I'd create a new one with `com.root-enterprise` instead of deleting the old one and recreating it on the new team.

These get created in the Apple developer portal

**Comment by atreat** [comment]
One important note is we will need to build the trip tracker from source off the client-side logging branch once this is merged.

We may want to hold off with this change until a release of the trip tracker is out that has the client side logging code

**Review by JesseACooper — APPROVED** [review]
LGTM. I don't think we need to be too careful about merging, the demo apps are pretty purpose specific.

**Review by CraigDunton — APPROVED** [review]

**Pull request closed without merging**
